docs/index.rst

```diff
@@ -0,0 +1,23 @@
+django-htmx documentation
```

Comment:
django-htmx?

Reply:
Embarrassing! I copied the documentation from django-htmx as a starting point.

Reply:
Only this one's left now! Keep on keeping on!